.github/SUPPORT.md

```diff
@@ -0,0 +1,54 @@
+# Getting help with lintr
+
+Thanks for using lintr. Before filing an issue, there are a few places
+to explore and pieces to put together to make the process as smooth as possible.
+
+## Reproducible example
```

Comment:
Making a reproducible example?

Would be more in line with the other section headers.